webui/settings.py

```diff
@@ -25,7 +25,10 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('SECRET_KEY')
+# DO NOT REMOVE if you're setting the password with Ansible, so Django creates the auth_user table;
+#   otherwise use the hashed password.
+# See https://docs.djangoproject.com/en/dev/ref/settings/#secret-key
+SECRET_KEY = os.getenv('SECRET_KEY', 'DO NOT REMOVE')
```

Comment:
I am not sure it is good idea , currently if variable is not defined anywhere you will get error that you must define it so you will clearly know what is going wrong . with this default value you will calm down django and your env will work even without variable defined but you may get troubles to login because you don't know the password because you haven't explicitly set it anywhere

Reply:
> I am not sure it is good idea , currently if variable is not defined anywhere you will get error that you must define it so you will clearly know what is going wrong . with this default value you will calm down django and your env will work even without variable defined but you may get troubles to login because you don't know the password because you haven't explicitly set it anywhere

I agree.  Will update the MR for the Ansible deployment code to set some dummy value.